src/app/components/webinars-banner/index.js

```diff
@@ -0,0 +1,146 @@
+import { useState, useEffect } from "@wordpress/element";
+import { Button, Title } from "@newfold/ui-component-library";
+import { ArrowRightIcon, CalendarIcon, ClockIcon } from "@heroicons/react/24/outline";
+import { ReactComponent as WebinarsVector } from 'App/images/webinars-vector.svg';
+import { SectionContainer, SectionContent } from 'App/components/section';
+
+/**
+ * A component that displays the next monthly webinar.
+ * 
+ * The component will attempt to fetch the webinars data from the Hiive CDN.
+ * If the file is fetched successfully, the component will verify the required shape of the data.
+ * When the data is valid, the component will filter the webinars to only include next webinar.
+ * The component will then render the next webinar.
+ * 
+ * If any of the above fails the component will not render.
+ * 
+ * @returns JSX.Element
+ */
+const WebinarsBanner = () => {
+	const [webinars, setWebinars] = useState([]);
+	const [nextWebinar, setNextWebinar] = useState(null);
+
+	useEffect(() => {
+		fetchWebinars();
+	}, []);
+
+	const fetchWebinars = async () => {
+		const webinarsEndpoint = 'https://cdn.hiive.space/resources/bluehost-webinars.json';
+
+		const response = await fetch(webinarsEndpoint);
+		if (!response.ok) {
+			console.warn('Could not fetch webinars data.');
+			return;
+		}
+
+		const data = await response.json();
+		if (
+			data.hasOwnProperty('isActive') &&
+			data.isActive &&
+			data.hasOwnProperty('items') &&
+			Array.isArray(data.items) &&
+			data.items.length > 0
+		) {
+			setWebinars(data.items);
+		}
+	}
+
+	useEffect(() => {
+		setNextWebinar(getNextWebinar());
+	}, [webinars]);
+
+	const getNextWebinar = () => {
+		const currentDate = new Date();
```

Comment:
Should we use a timezone offset here? This returns UTC I believe and the Webinars are in EST it looks like. However, it looks like we're just using a date check and not a datetime? Do we still want to display a webinar if it's only a few hours away?

Reply:
I'll make sure to convert to utc then. I can remember that.